src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -24,25 +27,26 @@
     private static final int DAYS_IN_MS = 24 * 60 * 60 * 1000;
 
     /**
-     * Returns the {@code CommitContributionSummary} generated from aggregating the {@code commitResults}.
+     * Returns the {@code CommitContributionSummary} generated from aggregating the {@code commitResults}, where the
+     * time is given in {@code zoneId} timezone.
      */
     public static CommitContributionSummary aggregateCommitResults(
-            RepoConfiguration config, List<CommitResult> commitResults) {
+            RepoConfiguration config, List<CommitResult> commitResults, ZoneId zoneId) {
```

Comment:
I think we should place the `zoneId` as a new field within `RepoConfiguration`, instead of passing it directly as a parameter. Then to access the `zoneId`, can just call `config.getZoneId()`. This is to standardise with all the other types of configurations.

Reply:
Okay sure, I will tweak the design of the configuration.